src/Microsoft.Identity.Client/MsalError.cs

```diff
@@ -470,6 +470,7 @@ public static class MsalError
         /// <para>Mitigation [App development]</para> In the application you could either inform the user that there are network issues
         /// or retry later
         /// </summary>
+        [Obsolete("MSAL no longer throws this error - it will allow the HttpClient exceptions to propagate. App developers may write their own logic for detecting access to the network issues, for example by using Xamarin.Essentials. ")]
```

Comment:
Is this change related to a bug/issue around this area?  Curious what prompted the change.

Reply:
@MarkZuber I think this ->[592](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/592)